_docs/2.0.0/usage/spark-jobs-quick-start.md

```diff
@@ -7,8 +7,97 @@ categories:
     - 'usage'
 redirect_from: /docs/usage/spark-jobs-quick-start
 ---
+{% capture docs_path %}{{ site.baseurl }}/docs/{{ page.version }}{% endcapture %}
+
+## Prerequsites
 
 This quick start guide presumes that you have went through :
 
 - [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start) guide
 - [Data & Data Quality Quick Start]({{ docs_path }}/usage/data-quick-start) guide
+
+## Running with spark-submit
+
+### Running Standardization
+
+```shell
+<spark home>/spark-submit \
+--num-executors 2 \
+--executor-memory 2G \
+--master yarn \
+--deploy-mode cluster \
+--driver-cores 2 \
+--driver-memory 2G \
+--class za.co.absa.enceladus.standardization.StandardizationJob \
+spark-jobs_<build_version>.jar \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <report_version> \
+--raw-format <data_format> \
+--row-tag <tag>
+```
+
+where:
+
+- `dataset_name` is the name gaven the dataset per [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start) guide Guide
+- `dataset_version` is a version of the dataset to use, which should have the correct schema and all the conformance rules wanted
+- `report_date` represents the date on which the data landed in the HDFS (in raw) and also the next part of the data output path. So if in Menas raw was specified as `/path/on/hdfs/raw` and the input parameter `report_date` as `2020-12-24` as a report date then the path where standardization will look for input files will be `/path/on/hdfs/raw/2020/12/24`. For the final part we are missing the report versions.
+- `report_version` is the final part of the path on HDSF. With `report_date` we finished with `/path/on/hdfs/raw/2020/12/24/v<report_version>`. This is the location where standardization will look for raw data.
+- `raw-format` and its specifics. Raw format tells the standardization in which format the data is on the HDFS and what are the specifics. CSV might have header, XML has row tag, etc. Here in the example we use row-tag. For more options for different types and run parameters see our [run documentations](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run) or just run `--help`
+
+
+### Running Conformance
+
+```shell
+<spark home>/spark-submit \
+--num-executors 2 \
+--executor-memory 2G \
+--master yarn \
+--deploy-mode cluster \
+--driver-cores 2 \
+--driver-memory 2G \
+--class za.co.absa.enceladus.conformance.DynamicConformanceJob \
+spark-jobs_<build_version>.jar \
+--menas-auth-keytab <path_to_keytab_file> \
+--dataset-name <dataset_name> \
+--dataset-version <dataset_version> \
+--report-date <report_date> \
+--report-version <data_run_version>
+```
+
+Here nothing new is added for the quick run. Of course there might be special options which are all documented in the [run documentations](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run)
```

Comment:
```suggestion
Here, nothing new is added for the quick run. Of course, there might be special options which are all documented in the [run documentation](https://absaoss.github.io/enceladus/docs/2.0.0/usage/run)
```

I think we should only use _documentation_ in the document, the plular form of the word, albeit technically possible, is rarely used. (also elsewhere in the doc).